lib/papertrail/http_client.rb

```diff
@@ -156,6 +136,21 @@ def escape(s)
         '%' + $&.unpack('H2' * $&.bytesize).join('%').upcase
       }.tr(' ', '+')
     end
+
+    def attempt(tries=3, wait=nil)
+      return unless block_given?
+
+      attempts = 0
+      begin
+        yield
+      rescue => e
```

Comment:
We'll want to continue rescuing `SystemCallError, Net::HTTPFatalError`. I think it makes sense to encode that here in this method and not supplied as arguments. Maybe that means this method is really `#attempt_http` or the like.